protocol-definitions/custom/Custom.yaml

```diff
@@ -1515,6 +1515,18 @@ customTypes:
         type: int
         since: 2.8
         nullable: false
+      - name: maxDegree
+        type: int
+        since: 2.8
+        nullable: true
```

Comment:
`FixSizedTypes` fields cannot be nullable (see readme - I was surprised by that)

Reply:
The nullable int value is true or false doesn't impact the generated codec. Additionally, nullable int is already utilized in this file.  So I prefer to leave it as this to maintain alignment with the mono-repo documentation.

Reply:
`nullable: true` is ignored here (only for fixed sized types!) and works like `nullable: false`. Do not introduce misleading, ignored setting. If it is used elsewhere for such types, then it is not good either, but with quick search I did not find any such case. Do you have an example of that?

Reply:
0c1f574e56aea824fd1f55bb529f2f6ea4ddebec